src/toast/toaster.js

```diff
@@ -193,11 +204,7 @@ export class ToasterContainer extends React.Component<
         if (__BROWSER__) {
           return (
             <>
-              {ReactDOM.createPortal(
-                root,
-                // $FlowFixMe
-                document.body,
-              )}
+              <Layer key={'toast-layer'}>{root}</Layer>
```

Comment:
It would be nice to use `Layer` here, but in the current state, it will break applications with multiple layers since any later rendered layers will overlap toast notifications since the toasts container is added earlier. And that may not be the desired behavior.

Reply:
gotcha